src/igvfd/schemas/treatment.json

```diff
@@ -0,0 +1,206 @@
+                        }
+                    }
+                },
+                {
+                    "properties": {
+                        "treatment_type": {
+                            "enum": [
+                                "chemical"
+                            ]
+                        },
+                        "treatment_term_id": {
+                            "pattern": "^((CHEBI:[0-9]{1,7})|(NTR:[0-9]{2,8}))$"
+                        }
+                    }
+                }
+            ]
+        }
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "documents": {
+            "description": "Documents that describe the treatment protocol."
+        },
+        "amount": {
+            "title": "Amount",
+            "type": "number"
+        },
+        "amount_units": {
+            "title": "Amount units",
+            "type": "string",
+            "enum": [
+                "mg/kg",
+                "mg/mL",
+                "mM",
+                "ng/mL",
+                "nM",
+                "percent",
+                "μg/kg",
+                "μg/kg",
+                "μg/mL",
+                "μM"
+            ]
+        },
+        "duration": {
+            "title": "Time duration",
+            "type": "number"
+        },
+        "duration_units": {
+            "title": "Time duration units",
+            "type": "string",
+            "enum": [
+                "second",
+                "minute",
+                "hour",
+                "day"
+            ]
+        },
+        "pH": {
+            "title": "pH",
+            "type": "number"
+        },
+        "purpose": {
+            "title": "Purpose",
+            "type": "string",
+            "enum": [
+                "activation",
+                "agonist",
+                "antagonist",
+                "control",
+                "differentiation",
+                "de-differentiation",
+                "perturbation",
+                "selection",
+                "stimulation"
+            ],
+            "comment": "The intended purpose for treating the samples; Activation: treatment is known to activate a pathway in the biosample; Agonist: a substance which initiates a physiological response when combined with a receptor; Antagonist: a substance that interferes with or inhibits the physiological action of another; Control: treatment applied to a sample for control purposes; Differentiation: treatment that is applied to convert a less specialized cell to a mature cell; De-differentiation: treatment used to reprogram differentiated cells back to less determined cell states; Perturbation: treatment applied to the sample in order to study the effect of its application; Selection: treatment used to affect biosample in a way that can be used to distinguish cells and selected for in the downstream steps; Stimulation: treatment stimulates a cellular pathway."
```

Comment:
(1) Differentiation: treatment that is applied to convert a less specialized cell to a mature cell - I think should change to: `Differentiation: treatment that is applied to convert a less specialized cell to a more specialized cell`.
(2) Selection: treatment used to affect biosample in a way that can be used to distinguish cells and selected for in the downstream steps; should change into `Selection: treatment used to affect biosample in a way that can be used to distinguish cells and select for in the downstream steps;`. 
(3) Stimulation: treatment stimulates a cellular pathway. should be: `Stimulation: treatment apllied to stimulate a cellular pathway.`